prusti-utils/src/config.rs

```diff
@@ -1017,3 +1043,29 @@ pub fn cargo_command() -> String {
 pub fn enable_type_invariants() -> bool {
     read_setting("enable_type_invariants")
 }
+
+/// When enabled, prusti should return various Data structures that are
+/// used by prusti-assistant, such as a list of method calls,
+/// a list of all procedures to be verified, etc.
+pub fn show_ide_info() -> bool {
+    read_setting("show_ide_info")
+}
+
+/// Very similar to no_verify but should only be used by prusti-assistant
```

Comment:
I will add a comment and describe it in the dev-guide, but yes there are some important differences. 
It is needed because of issue #1261. My solution for that problem was to generate a fake error when we skip verification.
The problem with `no_verify` is that it's value is not just what we pass via the command line. If no-verify is set to false by the user, it will still evaluate to true when compiling dependencies, and if I start throwing fake errors at that point, this will not work. 
If issue #1261 is resolved, this flag and the fake errors will no longer be necessary.
Let me know if you still think creating this enum you described would be a good idea, but I don't think that this skip_verification variable should be in there since it's probably temporary.